app/views/layouts/_service_notification_banner.html.erb

```diff
@@ -0,0 +1,13 @@
+<% unless @service_notifications.empty? %>
+  <% @service_notifications.each do |service_alert| %>
+      <div class="govuk-width-container app-site-width-container">
+        <br>
```

Comment:
The `<br>` here it causing the main content to get bumped down the page. Because it's not inside the `<div class="service_banner">`, the `<br>` still shows when the adjacent `<div>` is hidden.

The effect is more pronounced when multiple notifications are showing and have been hidden. Whilst this is a pretty extreme example (it's never going to happen in real life!), here's a screenshot where the page is showing 10 notifications which have all been hidden – so you're seeing 10 `<br>`s pushing the page down.

<img width="1238" alt="Screenshot 2020-08-07 at 12 26 20" src="https://user-images.githubusercontent.com/7735945/89641298-49f3ab80-d8a9-11ea-830e-deb03948ba04.png">

Ideally hidden banners would take up no extra space.

### Using `<br>` vs. CSS for spacing

If the `<br>` is used here to make sure there's spacing between the service banner and whatever it's close to, it might make more sense to use CSS `margin` to space it out. Using `<br>` for this feels a bit hacky, because it's not 'semantic HTML' – i.e. the `<br>` isn't actually representing a line-break in some content, but is instead being used for styling/spacing purposes.

Applying a CSS margin to the `.service_banner` element might actually solve both problems at once – CSS margins won't apply on hidden elements, and it'll be more semantic than using `<br>` HTML elements.